pkg/3scale/amp/operator/backend_options_provider.go

```diff
@@ -204,6 +210,12 @@ func (o *OperatorBackendOptionsProvider) workerPodTemplateLabels() map[string]st
 		labels[k] = v
 	}
 
+	if o.apimanager.Spec.Backend.WorkerSpec.Labels != nil {
```

Comment:
this `if` is not needed

Reply:
Addressed in this [commit](https://github.com/3scale/3scale-operator/pull/835/commits/f903bf41523372ad2297594e4efac295a12c0665)